app/helpers/application_helper/toolbar/container_node_center.rb

```diff
@@ -20,8 +20,7 @@ class ApplicationHelper::Toolbar::ContainerNodeCenter < ApplicationHelper::Toolb
           'product product-monitoring fa-lg',
           N_('Show Capacity & Utilization data for this Node'),
           N_('Utilization'),
-          :url       => "/show",
-          :url_parms => "?display=performance"),
+          :url => "/_performance"),
```

Comment:
Zita - it is not ready for review yet - I'll let you know when it is  -not all my changes are in